cadc-gms/src/intTest/resources/config/cadc-registry.properties

```diff
@@ -0,0 +1,10 @@
+## really temporary intTest setup so pdowler can check that code still works
+## NOT expected to work for anyone else
+ca.nrc.cadc.reg.client.RegistryClient.baseURL = https://haproxy.cadc.dao.nrc.ca/reg
+
+# configure LocalAuthority lookups
+## SRC IAM prototype 
+ivo://ivoa.net/sso#OpenID = https://ska-iam.stfc.ac.uk/                                                                                       
+
+http://www.opencadc.org/std/posix#group-mapping-0.1 = ivo://opencadc.org/src/posix-mapper
```

Comment:
Is the `user-mapping-0.1` intentionally missing here?  The `StandardIdentityManager` changes below only look for that, never the `group-mapping-0.1`.

Reply:
strange - don't know how that test managed to pass because it should be user-mapping-0.1

fixing

Reply:
I will fix the intTest to find the test token in classpath later